packages/contractkit/src/wrappers/BondedDeposits.ts

```diff
@@ -81,9 +93,9 @@ export class BondedDepositsWrapper extends BaseWrapper<BondedDeposits> {
     const notified = await this.getNotifiedDeposits(account)
     const weight = await this.getAccountWeight(account)
 
-    let gold = new BN(0)
-    bonded.forEach((bond) => (gold = gold.add(bond.value)))
-    notified.forEach((bond) => (gold = gold.add(bond.value)))
+    let gold = new BigNumber(0)
+    bonded.forEach((bond) => (gold = gold.plus(bond.value)))
```

Comment:
map/reduce? up to you though as sometimes it's uglier

Reply:
sure. I didn't see this. I was just changing Bn to BigNumber